guides/source/configuring.md

```diff
@@ -637,7 +637,7 @@ Is a boolean value which controls whether or not timestamping of `create` and `u
 
 #### `config.active_record.partial_inserts`
 
-Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default). The default value is `true`.
+Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default).
```

Comment:
IMO, we should prefer to explicitly mention what would be the default value if not set manually. Other all configs docs are added in a similar fashion. 😊 

```suggestion
Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default). Defaults to `false`.
```

Reply:
@abhaynikam 
thank you. I seem to have misunderstood the composition of the guide. Fixed this.

Reply:
@soartec-lab You did not misunderstand; you were correct.  When the default value depends on the `config.load_defaults` version, we omit the value from the description to prevent confusion like in #39387.

What's missing is just an entry under the "Baseline defaults:" heading like: 

```
* `config.active_record.partial_inserts`: `true`
```

Reply:
@jonathanhefner 
Ok, I've cleared up my misunderstandings.
I removed the previous change, the description of the default value. Also added to `Baseline defaults`.
could you review ?